src/DDSPreview.py

```diff
@@ -328,7 +332,8 @@ def settings(self):
                 mobase.PluginSetting("background r", self.__tr("Red channel of background colour"), 0),
                 mobase.PluginSetting("background g", self.__tr("Green channel of background colour"), 0),
                 mobase.PluginSetting("background b", self.__tr("Blue channel of background colour"), 0),
-                mobase.PluginSetting("background a", self.__tr("Alpha channel of background colour"), 0)]
+                mobase.PluginSetting("background a", self.__tr("Alpha channel of background colour"), 0),
+                mobase.PluginSetting("transparency", self.__tr("If enabled, transparency will be displayed."), True)]
```

Comment:
It'd be more technically accurate to refer to it as alpha rather than transparency or translucency. The whole reason this PR is necessary is that it's *not* always transparency data.